### PR TITLE
New Package(gh-121099): FireDaemon.OpenSSL 3.1.3

### LINKED ISSUE #121099: [Package Request]: OpenSSL
### How can we help?

I would like help so I can submit the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://kb.firedaemon.com/support/solutions/articles/4000121705#OpenSSL-Windows-Installer
Publisher: https://www.firedaemon.com/
Package Name: FireDaemon-OpenSSL-x64
Description: An installer for OpenSSL, different to the one by ShiningLight which is already available in WinGet but a p-i-t-a to maintain
Package Version: 3.1.3 (1.1.1w would be available as well)
Installer URL: https://download.firedaemon.com/FireDaemon-OpenSSL/FireDaemon-OpenSSL-x64-3.1.3.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
Resolve https://github.com/microsoft/winget-pkgs/issues/121099 .

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Have you tested your manifest in Windows Sandbox with `Tools/SandboxTest.ps1 -Manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121906)

### CODE REVIEW

File: manifests/f/FireDaemon/OpenSSL/3.1.3/FireDaemon.OpenSSL.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://download.firedaemon.com/FireDaemon-OpenSSL/FireDaemon-OpenSSL-x64-3.1.3.exe
+  InstallerSha256: C897942F2F319E070DCD0D882149E40B700ACAAA27532CC31439E172B5840DF1
+  InstallModes:
+  - silent
+  InstallerSwitches:
+    Silent: /exenoui /exelog fdopenssl3.log /qn /norestart REBOOT=ReallySuppress
```
Comment (Trenly) [inline manifests/f/FireDaemon/OpenSSL/3.1.3/FireDaemon.OpenSSL.installer.yaml:15]: The log parameter should be moved into the `Log` installer switch
```suggestion
    Silent: /exenoui /qn
    Log: /exelog <LOGPATH>
    Custom: /norestart REBOOT=ReallySuppress
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
Weird, the changes shouldn't have caused an unattended failure, since the silent and custom switches would be applied to all runs

**Comment by Dragon1573** [comment]
The previous `Silent` switch was copied from [the User Guide](https://kb.firedaemon.com/support/solutions/articles/4000121705#OpenSSL-Windows-Installer). Can't figure out what was wrong...

File: manifests/f/FireDaemon/OpenSSL/3.1.3/FireDaemon.OpenSSL.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://download.firedaemon.com/FireDaemon-OpenSSL/FireDaemon-OpenSSL-x64-3.1.3.exe
+  InstallerSha256: C897942F2F319E070DCD0D882149E40B700ACAAA27532CC31439E172B5840DF1
+  InstallModes:
+  - silent
+  InstallerSwitches:
+    Silent: /exenoui /qn
+    Log: /exelog <LOGPATH>
```
Comment (Trenly) [inline manifests/f/FireDaemon/OpenSSL/3.1.3/FireDaemon.OpenSSL.installer.yaml:16]:
```suggestion
    SilentWithProgress: /exenoui /qb
```
This should fix it (hopefully)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**